docs/go-logging-instrumentation.md

```diff
@@ -0,0 +1,113 @@
+---
+id: go-logging-instrumentation
+title: Scope Go Agent Logging Instrumentation
+sidebar_label: Logging Instrumentation
+---
+
+The Scope Go agent has support for capturing logs from the following sources:
+
+- Test logs (calls to `t.Log()` and other test methods) are automatically instrumented using monkey patching. Includes support for capturing source file and number information, and also for the exact timestamp and level of the log event. It is compatible with parallel tests.
+- Scope Test logs (calls to `test.Log` and other test methods) includes support for capturing source file and number information, and also for the exact timestamp and level of the log event. It is compatible with parallel tests.
```

Comment:
I'd remove this line here, and just leave the explanation in the `Scope Test logs` section